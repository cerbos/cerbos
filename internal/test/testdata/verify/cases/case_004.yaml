description: Invalid test suite with non-yaml data

