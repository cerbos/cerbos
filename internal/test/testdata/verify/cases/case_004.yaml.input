-- bad_test.yaml --
x
