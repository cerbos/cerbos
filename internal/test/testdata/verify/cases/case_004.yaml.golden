{
  "suites": [
    {
      "file": "bad_test.yaml",
      "name": "Unknown",
      "summary": {
        "overallResult": "RESULT_ERRORED"
      },
      "error": "file is not valid: { /: [expected object, but got string] }"
    }
  ],
  "summary": {
    "overallResult": "RESULT_ERRORED"
  }
}
